### PR TITLE
Upgrade k8s to 1.0.3 for ubuntu

### PR DESCRIPTION
**Pull request by dalanlan**
Test passed locally.
Refactor the guide for readability.

### CODE REVIEW

**Comment by piosz** [comment]
@k8s-bot ok to test

File: docs/getting-started-guides/ubuntu.md [hunk 1]
```diff
@@ -35,53 +35,71 @@ Kubernetes Deployment On Bare-metal Ubuntu Nodes
...
+in the given examples. You can scale to **any number of nodes** by changing some settings with ease.
+The original idea was heavily inspired by @jainvipin 's ubuntu single node
+work, which has been merge into this document.
 
 [Cloud team from Zhejiang University](https://github.com/ZJU-SEL) will maintain this work.
 
 ## Prerequisites
 
-*1 The nodes have installed docker version 1.2+ and bridge-utils to manipulate linux bridge*
+1 The nodes have installed docker version 1.2+ and bridge-utils to manipulate linux bridge.
```
Comment (piosz) [inline docs/getting-started-guides/ubuntu.md]: How about making it list by adding dot after numbers (see https://guides.github.com/features/mastering-markdown/#examples tab Lists)

**Comment by piosz** [comment]
Just a nit, otherwise lgtm.

Thank you @dalanlan for the contribution!

**Comment by dalanlan** [comment]
The comment has been addressed.
FYI @piosz

**Pull request merged**